### ISSUE TITLE
how can i change the regex for url matching for slug name

### ISSUE THREAD

**Issue by sushovan23** [llazzaro/django-scheduler#266]
Hello,
I have integrated your application and it's working great.
I came across an issue which is mainly because my slug names also contain numericals and spaces.

In my own application view and have made the slug url as: ?P<calendar_slug>[\w-]. instead of ?P<calendar_slug>[-\w]+ which is used everywhere in urls.py of schedule. Could you let me know a way where I can change the urls in my application or do I need to change in your app in my environment?

**Comment by llazzaro** [comment]
Yes, avoid including the schedule urls and use a custom one. The first Idea I have is to just copy the https://github.com/llazzaro/django-scheduler/blob/develop/schedule/urls.py to your project and fix the urls, then include that urls_schedule.py to your project.
Your are free to add urls like you want like this:
```
 url(r'^calendar/year/(?P<calendar_slug>[\w-])/$',
        CalendarByPeriodsView.as_view(template_name='schedule/calendar_year.html'),
        name="year_calendar",
        kwargs={'period': Year}),
```

Good luck
